fe2-android/app/src/main/java/com/github/dedis/student20_pop/home/fragments/HomeFragment.java

```diff
@@ -60,7 +60,7 @@ public void onActivityCreated(@Nullable Bundle savedInstanceState) {
     setupListAdapter();
     setupListUpdates();
 
-    mHomeViewModel.setupDummyLAO();
+    //    mHomeViewModel.setupDummyLAO();
```

Comment:
The `setupDummyLAO` method has been deleted, I think it's better if we delete this as well